### PR TITLE
Adding 'es' Tool for Renewal translation

### PR DESCRIPTION
**Pull request by lh1008**
Hello everyone,

Adding Spanish "Tool for Renewal" translation. It's ready to be reviewed. Pinging @BlackLotus64 

English version:

- Website:
  - https://www.monerooutreach.org/stories/the-tool-for-renewal.html

- Repository:
  - https://github.com/monero-ecosystem/outreach-docs/blob/master/monero-outreach-docs/en/storyboard/tool_for_renewal_en.md

Pinging @erciccione

### CODE REVIEW

**Comment by BlackLotus64** [comment]
Sorry, I saw this before, have been a little busy and adjusting to some changes in life. Will try to review it this weekend.

**Comment by BTCLovera** [comment]
Ortografía:

Espacio 12 - croncretamente - concretamente

Espacio 18 - declaró con tilde

Espacio 26 - "Te motivamos Ha que hagas" Ha van sin h? por preterito y el verbo hacer?

Espacio 58 - Empoderar lo cambiaría por "permitierles" o "capacitar"

Espacio 58 - "Corten de la ecuación" —- No seria mejor "eliminen/saquen de la ecuacion"

File: monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+Los sistemas financieros a menudo están inextricablemente vinculados a los disturbios civiles y al cambio social. Las bases de la Revolución Americana incluyeron el deseo de la colonia por controlar los impuestos y el destino económico.  
+
+A medida que los bancos centrales comenzaron a imprimir dinero después del desmantelamiento total del patrón oro en 1971, nace un nuevo sistema monetario por decreto o sistema fiat que se convertiría en el eje monetario global, el dólar estadounidense (USD) determinaría el valor de la mayoría de las otras monedas. Las personas ahora están viendo las consecuencias indeseadas de este sistema financiero, croncretamente hablando de la fragilidad socioeconómica, la inequidad y la disparidad. Las persona quieren algo mejor que equilibre el poder. Esta vez, no necesitamos una guerra para lograrlo. Podemos optar simplemente salirnos pacíficamente escogiendo usar un sistema financiero tecnológico que está siendo desarrollado por y para las personas.
```
Comment (BTCLovera) [inline monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md:12]: ...este sistema financiero, "croncretamente" hablando..

Concretamente

File: monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+Sin embargo, al ser completamente nuevo, Bitcoin tenía fallas técnicas, incluyendo mala privacidad. Y así, otra persona o grupo seudónimo llamado Nicolas van Saberhagen desarrollaron el protocolo CryptoNote. Los programadores se dieron cuenta y nació Monero. Monero resolvió muchos de los inconvenientes de Bitcoin, incluyendo su fácil vigilancia por parte de gobiernos y empresas. Tal como declaro la compañía de análisis de blockchain Chainalysis recientemente en un seminario web público, [chainalysis.com/.../Advanced-Obfuscation-Techniques-Public-Webinar.mp4font](https://go.chainalysis.com/rs/503-FAP-074/images/Advanced-Obfuscation-Techniques-Public-Webinar.mp4), “Monero está mucho más enfocado en la privacidad que sus semejantes. Todo en Monero está ofuscado, no se puede distinguir nada.
```
Comment (BTCLovera) [inline monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md:18]: ...tal como declarO la ....
declaró

File: monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+Sea que necesites mover tu dinero a alguna reserva de valor privada por necesidad o quieras por principios optar salirte del actual sistema financiero, únete a nosotros. Juntos podemos usar nueva tecnología para limitar la intromisión de gobiernos y bancos corruptos, y participar en un sistema financiero que empodere a las personas. Estamos todos en esto. Usemos estas herramientas para ser el cambio que queremos ver para nosotros, las futuras generaciones y el mundo.  
+
+No esperamos que tomes nuestra palabra para nada de esto y te motivamos ha que hagas tus propias investigaciones y revises con fuentes. Compartiremos algunas referencias que recomendamos por donde comenzar. Una vez hayas terminado de hacer tu propia investigación, vuelve para ver la guía rápida de como comenzar a usar Monero (ver más abajo).
```
Comment (BTCLovera) [inline monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md:26]: "Te motivamos Ha que hagas" 

Ha van sin h? por preterito y el verbo hacer?

File: monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+- [https://web.getmonero.org/es/get-started/mining/](https://web.getmonero.org/es/get-started/mining/)  
+- [Minar Monero para apoyar la red](https://www.monerooutreach.org/mejores_practicas_monero/mina-para-apoyar-la-red.html)  
+- [https://github.com/xmrig/xmrig](https://github.com/xmrig/xmrig)  
+
+**Crypto ATMs.** Puedes comprar XMR sin un banco utilizando dinero en efectivo o tarjetas de crédito en los cajeros ATMs de criptomonedas alrededor del mundo. Estos comunmente vienen con comisiones altas pero son convenientes. Para ubicar uno cerca de ti, usa un servicio como [CoinAMTRadar.com](https://coinatmradar.com/) y filtra el mapa por monedas (Coins): Monero (XMR) y operaciones (Operations): Comprar (Buy).  
+
+**Aceptando Monero (para negocios).** La comunidad global de desarrollarodes de Monero ha creado herramientas para empoderar a los negocios a aceptar XMR para sus productos y servicios. Corten de la ecuación al costoso intermediario y permitanle a sus clientes pagar fracciones de centavos por comisiones en transacciones para usted. Aprende más:
```
Comment (BTCLovera) [inline monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md:58]: Empoderar lo cambiaría por "permitierles" o "capacitar"

Comment (BTCLovera) [reply monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md:58]: - "Corten de la ecuación" —- No seria mejor "eliminen/saquen de la ecuacion

File: monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md (hunk 1 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md:12]: Do you want me to put double quotes to the word "_concretamente_"?

File: monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md (hunk 2 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md:18]: Fixed

File: monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md (hunk 4 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md:58]: Yeah you're right. I made the changes. Fixed :)

File: monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md (hunk 3 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md:26]: Yes you're right. "a" is not with an "h". Removed. Fixed.

File: monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md (hunk 1 above)
Comment (lh1008) [reply monero-outreach-docs/translations/es/storyboard/tool_for_renewal_es.md:12]: Oh sorry, I see it's a typo. Will fix it right away.

**Comment by lh1008** [comment]
Thank you @BTCLovera. Pinging @BlackLotus64 if you were able to check it out.

**Comment by lh1008** [comment]
It seems there are no more comments.

@erciccione this should be ready to merge.

**Pull request merged**